Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Controllers/ProductsController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Know_Your_Nation_Speedy.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace Know_Your_Nation_Speedy.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ProductsController : ControllerBase
```

Comment:
Suggestion rename it to ProductController. 

If you think about it you only working with a product and not multiple products.